packages/tradershub/src/screens/PersonalDetails/Sections/Details.tsx

```diff
@@ -15,16 +17,16 @@ const Details = () => {
             <Text as='p' className='my-800' weight='bold'>
                 Details
             </Text>
-            <div className='outline outline-1 outline-system-light-active-background mx-800 p-800 rounded-400 pb-2000'>
+            <div className='outline outline-1 outline-system-light-active-background md:mx-800 p-800 rounded-400 pb-2000'>
                 <InlineMessage className='items-start mb-800' variant='warning'>
-                    <Text>
+                    <Text size={isMobile ? 'sm' : 'md'}>
```

Comment:
tailwind tokens doesn't have `font-size: 10px`, so using `isMobile` for the prop for now